ci/gpu/build.sh

```diff
@@ -59,8 +59,8 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
 # Install the master version of dask, distributed, and dask-ml
 gpuci_logger "Install the master version of dask and distributed"
 set -x
-pip install "git+https://github.com/dask/distributed.git@2021.07.1" --upgrade --no-deps
-pip install "git+https://github.com/dask/dask.git@2021.07.1" --upgrade --no-deps
+pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-deps
+pip install "git+https://github.com/dask/dask.git@main" --upgrade --no-deps
```

Comment:
Wonder if we should do something like this ( https://github.com/rapidsai/dask-cuda/pull/196/commits/855eeac89fe13448f74d385eb7ba9cd89b5aa2fe ) to simplify this process